src/main/java/de/zalando/aruha/nakadi/service/EventStreamConfig.java

```diff
@@ -7,6 +7,9 @@
 
 @Immutable
 public class EventStreamConfig {
+
+    private static int BATCH_FLUSH_TIMEOUT_DEFAULT = 30;
```

Comment:
I don't like that we have this constant in two places now: here and additionally in controller method parameter annotation. But I can't offer a better solution now :)

The only possibility I can think about is to move all default values from controller-method to EventStreamConfig class. But for that we will need to change all parameters of controller-method to be Objects instead of primitives (Integer instead of int etc.) - because we will need them to differ between 0 and null values (if we use primitives that will not be possible)

What do you think?